packages/block-library/src/group/edit.js

```diff
@@ -25,19 +56,33 @@ function GroupEdit( { attributes, setAttributes, clientId } ) {
 		},
 		[ clientId ]
 	);
+	const { tagName: TagName = 'div', templateLock, layout = {} } = attributes;
+	const { contentSize, wideSize } = layout;
 	const blockProps = useBlockProps();
-	const { tagName: TagName = 'div', templateLock } = attributes;
-	const innerBlocksProps = useInnerBlocksProps(
-		{
-			className: 'wp-block-group__inner-container',
-		},
-		{
-			templateLock,
-			renderAppender: hasInnerBlocks
-				? undefined
-				: InnerBlocks.ButtonBlockAppender,
-		}
+	/* TODO: find a way to render this extra div as a child of the inner block wrapper 
+	const extraChildren = (
+		<BoxControlVisualizer
+			values={ attributes.style?.spacing?.padding }
+			showValues={ attributes.style?.visualizers?.padding }
+		/>
 	);
+	*/
+	const innerBlocksProps = useInnerBlocksProps( blockProps, {
+		templateLock,
+		renderAppender: hasInnerBlocks
+			? undefined
+			: InnerBlocks.ButtonBlockAppender,
+		__experimentalLayout: {
+			type: 'default',
+			// TODO: only pass this if the theme supports the new alignments.
+			// otherwise make all alignments avaiable.
+			// Find a way to inject this using the support flag.
```

Comment:
This is the only thing that I was not able to inject using a block support flag automatically, I wonder if we can do this somehow. cc @ellatrix